build.sh

```diff
@@ -30,19 +27,16 @@ function build_grub() {
 }
 
 function build_qemu() {
-  # Dependencies
-  if [ -x "$(command -v apt)" ]; then
-    sudo apt -qq install git \
-                         zlib1g-dev \
-                         libglib2.0-dev libfdt-dev libpixman-1-dev \
-                         libsdl2-dev libvulkan-dev
-  fi
   # Building
   cd ${path_orbital}
   cd ${path_qemu}
-  ./configure --target-list=ps4-softmmu \
-    --enable-sdl --enable-vulkan --enable-debug --disable-capstone \
-    --enable-hax
+  
+  local flags="--target-list=ps4-softmmu --enable-sdl --enable-vulkan --enable-debug --disable-capstone --enable-hax "
```

Comment:
I'd prefer if you simply kept the original `./configure ...` command and simply added the `${additional_flags}` afterwards:
```diff
   ./configure --target-list=ps4-softmmu \
     --enable-sdl --enable-vulkan --enable-debug --disable-capstone \
-    --enable-hax
+    --enable-hax ${additional_flags}
```